libraries/Microsoft.Bot.Builder.Dialogs.Adaptive/Schemas/Actions/Microsoft.BeginDialog.schema

```diff
@@ -22,11 +22,11 @@
             "oneOf": [
                 {
                     "$kind": "Microsoft.IDialog",
-                    "pattern": "^(?!(=)).*",
                     "title": "Dialog"
                 },
                 {
                     "$ref": "schema:#/definitions/equalsExpression",
+                    "pattern": "^(?!(=)).*",
```

Comment:
No, this should be
{
                    "$ref": "schema:#/definitions/stringExpression",
                    "examples": [
                      "dialog13",
                        "=settings.dialogId"
                    ]
                }